packages/http.go

```diff
@@ -59,17 +58,24 @@ func ServeFile(w http.ResponseWriter, r *http.Request, p *Package, name string)
 		return
 	}
 
-	f, err := fs.Open(name)
+	stat, err := fs.Stat(name)
 	if os.IsNotExist(err) {
 		http.Error(w, "resource not found", http.StatusNotFound)
 		return
 	}
+	if err != nil {
+		log.Printf("failed to check file modification time (%s) in package: %v", name, err)
```

Comment:
This log line might be misleading. I would just say that `stat failed: <err>`.

Reply:
Thanks, changed, keeping the requested path in the error message.